src/lib/geo_location_utils.js

```diff
@@ -51,7 +51,7 @@ function countryNameToISO3(countryName) {
         var iso3 = countryIds[i],
             regex = new RegExp(countryRegex[iso3]);
 
-        if(regex.test(countryName.toLowerCase())) return iso3;
+        if(regex.test(countryName.trim().toLowerCase())) return iso3;
```

Comment:
In case someone is wondering, `.trim` works in IE9

![image](https://cloud.githubusercontent.com/assets/6675409/23080713/d43162cc-f51f-11e6-812b-ea8b3e55678a.png)

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/trim